.gitignore

```diff
@@ -1,3 +1,4 @@
 .env
 node_modules/
 dist/
+config_local.js
```

Comment:
Not sure why you remove it in the room `.gitignore`.

I think I would either add `src/config_local.js` to this file, or create a `.gitignore` in the `src/` directory with `config_local.js`